ga4gh/server/datamodel/bio_metadata.py

```diff
@@ -146,15 +146,15 @@ def toProtocolElement(self):
                 gaIndividual.info[key].values.add().string_value = value
         return gaIndividual
 
-    def populateFromRow(self, row):
+    def populateFromRow(self, ind):
```

Comment:
I prefer using the full name... otherwise, what is `ind`?  Index?  Indent?  Indicator?  Individual?  It's nice to have context provided.  A bit more typing, yes, but I think it's worth it, and it's the style that we have in our code hitherto.